common/app-starters-security-common/README.adoc

```diff
@@ -1,10 +1,15 @@
 === `App Starters Security` Common Module
 
-This artifact contains a Spring Boot auto-configuration to disable security in the application that set `spring.cloud.stream.security.enabled = false` (default).
-If the `spring.cloud.stream.security.enabled` property is explicitly set to `true` then this configuration falls back to the default Spring Security configuration.
+Spring Boot auto-configuration to manage the web security of the application starters.
+
+It also provides default configuration for web security when the actuator dependency is on the classpath. By default it allows unauthenticated access to the `HealthEndpoint`, `LogFileWebEndpoint`, `InfoEndpoint` and `BindingsEndpoint`.
```

Comment:
One sentence per line: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
What is your question?
You have a period after `classpath` word and then you start a new sentence immediately on the same line.

I'm not sure what and how to explain yet, but would be great to have consistency even with docs.

Sorry if I sound rude...